app/src/main/res/values/styles.xml

```diff
@@ -176,4 +168,10 @@
         <item name="android:gravity">center_vertical</item>
     </style>
 
+    <style name="collapsedAppBar" parent="@android:style/TextAppearance.Medium">
+        <item name="android:textSize">18sp</item>
```

Comment:
Also, I think you may need to use a separate style for the place suggestion screen, because now the text starts out small (18sp) and then grows to 20sp when you collapse the toolbar, which seems backwards.